SPECS/keepalived/keepalived.spec

```
@@ -45,14 +51,26 @@ healthchecks and LVS directors failover.
 autoreconf -f -i
 %configure \
     --with-systemdsystemunitdir=%{_unitdir} \
+    --disable-nftables  \
     --enable-snmp       \
-    --enable-snmp-rfc
+    --enable-snmp-rfc   \
+    --with-init=systemd
 make %{?_smp_mflags} STRIP=/bin/true
 
 %install
+rm -rf %{buildroot}
 make install DESTDIR=%{buildroot}
```

No need for that with the way we build packages.

```suggestion
make install DESTDIR=%{buildroot}
```